### DISCUSSION TITLE
Render a list for both html and pdf format?

### DISCUSSION THREAD

**Discussion by tony-an-phd** [quarto-dev/quarto-cli#11319] [Q&A] [answered]
### Description

When I specify both html and pdf as output format, I write a list as 
```
- item one
- item two 
```
etc.. and it rendered just as "- item one" in both html and pdf, instead of the normal bullet + indentation for a usual list. So I decided to use `<ul> <li> </ul>`as the markdown for html, and it looks correctly in the html output, but the pdf will just print these tags as plain text. So how do I make a list that can make both output rendered correctly?

**Comment by mcanouil** [comment]
Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us to recreate your problem so that we can fix it: please help us help you! Thanks.

<img width="1624" alt="image" src="https://github.com/user-attachments/assets/4b626bda-055c-4b72-b2cb-eb2136df8cc5">


````qmd
---
title: "Quarto Playground"
format: pdf
keep-tex: true
---

- item one
- item two

````

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by tony-an-phd** [reply]
I am attaching the qmd and it's rendered pdf, and the screen shot of html. Thanks. 
![list html](https://github.com/user-attachments/assets/b3a8fb10-139b-4fff-82dd-60aa59a34b16)
[list.pdf](https://github.com/user-attachments/files/17639752/list.pdf)

````qmd
---
title: "how a test is rendered for both html and pdf"
author: 
  - name: "Tony"
date: "November 2024"
format:
  html:
    theme: default
    toc: true 
    toc-depth: 2 
    
  pdf:
    documentclass: article
    toc: true
    toc-depth: 2
    number-sections: true
---

# Chapter 1

The following list should be indented and labeled with a bullet

    - item 1
    - item 2

But they don't in both html and pdf outputs. 

The following uses the html tags:

<ul> 
    <li> item 1 using html listing tag
    <li> item 2 using html listing tag
</ul>

The result works fine in html output but not in pdf. 

# References
````
``` quarto check
Quarto 1.6.30
[>] Checking environment information...
      Quarto cache location: C:\Users\ynots\AppData\Local\quarto
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.4.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.46.3: OK
      Typst version 0.11.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.6.30
      Path: C:\Program Files\Quarto\bin
      CodePage: 1252

[>] Checking tools....................OK
      TinyTeX: v2024.11
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Using: TinyTex
      Path: C:\Users\ynots\AppData\Roaming\TinyTeX\bin\windows\
      Version: 2024

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[>] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/
```

**Reply by mcanouil** [reply]
That's not how Markdown works.

Your syntax is invalid thus not working.

Please read the guide on authoring using Markdown: https://quarto.org/docs/authoring/markdown-basics.html

**Reply by tony-an-phd** [reply]
Well, that's I got the example, I tried used `*` first actually, it just printed a `*`, so I asked chatGTP and it suggested me to use `-`, and it didn't work, so it suggested me to use the html tag. lol So, what's the valid syntax then?

**Reply by mcanouil** [reply]
There are no examples of first level markdown list that are indented.
Look at the code I provided and look at the examples from the guide.

**Reply by tony-an-phd** [reply]
Besides not indented, they are not changed into bullet. Let's forget about my code. Can you provide an example to your code that produces both html and pdf that rendering the list the same way?

**Reply by mcanouil** [reply]
Take the code I shared earlier and add the HTML format.

This is wrong:

```md
    - item 1
    - item 2
```

This is correct and documented:

```md
- item 1
- item 2
```

Look how it is coloured differently even here on GitHub.

**Comment by tony-an-phd** [comment, marked as answer]
Never mind. I used the markdown-mode in emacs and it automatically indented the listed items in the qmd file, and those indentations in qmd are translated into `\begin{verbatim}` instead of `\begin{itemize}` in tex. Once I removed those indentation, they are rendered both correctly in pdf and html output.
